tests/PHPStan/Rules/Methods/CallMethodsRuleTest.php

```diff
@@ -104,70 +104,6 @@ public function testCallMethods(): void
 				63,
 				'Learn more at https://phpstan.org/user-guide/discovering-symbols',
 			],
-			[
```

Comment:
Do not remove tests from rules affected by this change, instead fix the rules so that these problems are still reported ;)

Reply:
But isn't this the expected behaviour change?

Reply:
No, you still want to know that you're calling a `void` function and you are not supposed to read the result.

Reply:
OK, that makes more sense to me and king of answers my open questions. Will need to think about the implementation again then though :)